Dockerfile-origin-release

```diff
@@ -9,8 +9,8 @@ RUN curl -sLk https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
     rm -f /tmp/jq-linux64
 
 COPY --from=origin-release release-manifests/image-references .
-RUN cat image-references
 RUN jq '.spec.tags |= map(.name as $name | if (['$IMAGES'] | index($name)) then ("'$IMAGE_ORG'/origin-"+$name+":latest") as $override | ("Switching \($name): \(.from.name) => \($override)" | stderr) as $stderr | .from.name |= $override else . end)' image-references > image-references-updated
+RUN diff -u image-references image-references-updated || true
```

Comment:
what is this?

Reply:
just syncing with other operator repos, @tnozicka ?